POM of com.gluonhq:ignite-micronaut:jar:1.1.0 (in Maven Central) is invalid

Hello,

I try to use this artifact as dependency in a project's `pom.xml`.
Get the following warning and error message running Maven in 'debug mode' (`-X`).
```xml
[WARNING] The POM for com.gluonhq:ignite-micronaut:jar:1.1.0 is invalid, transitive dependencies (if any) will not be available: 1 problem was encountered while building the effective model for com.gluonhq:ignite-micronaut:1.1.0
[ERROR] 'dependencies.dependency.version' for javax.annotation:javax.annotation-api:jar is missing. @
```

--- a/micronaut/build.gradle
+++ b/micronaut/build.gradle
@@ -9,7 +9,7 @@ dependencies {
     implementation "io.micronaut:micronaut-inject:$micronautVersion"
     implementation("io.micronaut:micronaut-validation:$micronautVersion")
     implementation("io.micronaut:micronaut-runtime:$micronautVersion")
-    implementation("javax.annotation:javax.annotation-api")
+    implementation "javax.annotation:javax.annotation-api:1.3.2"
     runtimeOnly("ch.qos.logback:logback-classic:1.2.3")
 
     compile project(':common')

